masterlist.yaml

```
@@ -8384,6 +8384,9 @@ plugins:
       - <<: *patchProvided
         subs: [ 'Knights of the Nine' ]
         condition: 'active("Knights.esp") and active("Knights - Unofficial Patch.esp") and not file("Oblivion Content Restoration Project -- Knights.esp")'
+      - <<: *wildEdits
+        subs: [ '[Cleaning instructions](https://github.com/MacSplody/xGuides/blob/master/index.md#-oblivion-content-restoration-project-)' ]
```

That's the link to the repo, the link to the website should be used instead:
```suggestion
        subs: [ '[Cleaning instructions](https://macsplody.github.io/xGuides/#-oblivion-content-restoration-project-)' ]
```